### PR TITLE
Generalize CC deployment scripts based on current folder structure an…

### PR DESCRIPTION
**Pull request by qfweiwei**
## Purpose

Generalize CC deployment scripts based on current folder structure and naming convention.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: custom-commands/scripts/azuredeploy.parameters.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "value": "test7resource-8070"
+    },
+    "functionName": {
+      "value": "test7resource-8070"
+    },
+    "storageName": {
+      "value": "test7resource8070"
+    }
+  }
+}
```
Comment (dargilco) [inline custom-commands/scripts/azuredeploy.parameters.json:18]: Please see one of my recent changes. I updated test7resource to generic name.

Comment (dargilco) [reply custom-commands/scripts/azuredeploy.parameters.json:18]: See https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant/blob/master/custom-commands/hospitality/deployment/azuredeploy.parameters.json

**Comment by chschrae** [comment]
It looks like there are a lot of line ending changes here. This makes it very hard to see what has actually changed. Is it possible to update this PR without the line ending changes?

File: custom-commands/scripts/azuredeploy.parameters.json (hunk 1 above)
Comment (qfweiwei) [reply custom-commands/scripts/azuredeploy.parameters.json:18]: OK. I will update azuredeploy.parameters.json accordingly.

**Comment by qfweiwei** [comment]
@chschrae Sorry for all the inconveniences. I used VS Code to re-format the code. Though it causes pain now, but could improve readability for long term. One workaround is to configure diff comparison to ignore line ending. Or, just have a quick look, I will merge changes from branch #364.

**Comment by chschrae** [comment]
@qfweiwei  vscode allows you to choose line endings. Formatting is fine but line endings don't do anything for reading so we should get in the habit of not changing them. Since it can break linux scripts

**Pull request merged**